charts/aad-pod-identity/templates/nmi-daemonset.yaml

```diff
@@ -70,6 +70,9 @@ spec:
           {{- if .Values.nmi.ipTableUpdateTimeIntervalInSeconds }}
```

Comment:
All chart changes are to made only in [manifest_staging](https://github.com/Azure/aad-pod-identity/tree/master/manifest_staging/charts/aad-pod-identity) directory. Could you make the changes to charts there and revert it from here?